### PR TITLE
Add highest_score field to HackProjectScoreCategory model

### PR DESCRIPTION
**Pull request by Sarosim**
S08 - Judging submissions
Score Categories can have different score ranges (e.g. 1-10, 1-15)
Adding this field sets the upper end of the scale, so the admin can define for each scoring category the scale by providing the highest possible score for that given category.

I'm just realising that this field was indeed on the original schema (with name score), but wasn't implemented in the models. In this respect it can be treated as a fix. 

This might affect other user stories so I'm submitting this PR without any other changes to S08, so others can pull it and start working with this fix.

Tests:
Added the field to the test case but no assertion is added for it. I run the tests and all pass.

Know bugs/errors: 
None

In my version, the field is named `highest_score`, but I can rename it, if you guys deem necessary.

image

### CODE REVIEW

File: hackathon/models.py [hunk 1]
```diff
@@ -166,6 +166,9 @@ class HackProjectScoreCategory(models.Model):
                                    on_delete=models.CASCADE,
                                    related_name="hackprojectscorecategories")
     category = models.CharField(default="", max_length=255)
+    # Score Categories can have different score range (e.g. 1-10, 1-15)
+    # this field sets the upper end of the scale
+    highest_score = models.IntegerField(default=10)
```
Comment (stefdworschak) [inline hackathon/models.py:171]: I'd say change it to `max_score` which is probably more fitting. `Highest score` sounds more like something related to a `highscore` to me.

**Review by stefdworschak — CHANGES_REQUESTED** [review]
@Sarosim some minor changes, but I would also add an assertion to test this is working.

File: hackathon/models.py (hunk 1 above)
Comment (Sarosim) [reply hackathon/models.py]: Thanks Stefan, obviously min_score` and `max_score` are the most meaningful names, corrected, tested with assertions, all tests passing.

**Comment by Sarosim** [comment]
Added `min_score` field with `default=1` and renamed `highest_score` to `max_score`, added assertions for both fields to the tests and run them. All tests passing

**Review by stefdworschak — APPROVED** [review]
LGTM

**Pull request merged**